walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,69 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Arrays;
+import java.util.ArrayList;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    if (request.getAttendees().size() <= 0) {
```

Comment:
can size() be less than 0?

Reply:
No, I just always do that with potential 0 length things, just to be safe. You never know lol

Reply:
It probably works but if you know size() returns a uint32 then you can be sure that it can never < 0.

Reply:
okay I'll change it to check if size==0 instead